packages/terra-divider/src/_SectionDivider.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import PropTypes from 'prop-types';
+import styles from './Divider.module.scss';
+
+const propTypes = {
+  /**
+   * String to be displayed inline with the divider.
+   * A level should also be used whenever using the text.
+   */
+  text: PropTypes.string.isRequired,
+  /**
+   * Sets the heading level to One of `3`, `4`, `5`, `6`. This helps screen readers to announce appropriate heading levels.
+   * Changing `level` will not visually change the style of the content.
+   */
+  level: PropTypes.oneOf([3, 4, 5, 6]).isRequired,
+  /**
+   * IDs should be used with 'aria-labelledby' to associate headings with the corresponding page area when needing to label regions.
```

Comment:
for this section divider, will there be a case where content will be provided by other element. or should the screenshot will always read out the text prop that is provided ?

Reply:
IDs [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) should be included if there is more than one focusable separator.

Reply:
Wording change:
```suggestion
   * IDs should be used with 'aria-labelledby' to associate headings with the corresponding list, section, or page area to label regions.
```